### PR TITLE
Include ALTO in list of supported output formats

### PR DESCRIPTION
**Pull request by jakesebright**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ and GitHub's log of [contributors](https://github.com/tesseract-ocr/tesseract/gr
 
 Tesseract has **unicode (UTF-8) support**, and can **recognize more than 100 languages** "out of the box".
 
-Tesseract supports **various output formats**: plain-text, hocr(html), pdf, tsv, invisible-text-only pdf.
+Tesseract supports **various output formats**: plain-text, hocr(html), pdf, tsv, invisible-text-only pdf, alto(xml).
```
Comment (stweil) [inline README.md]:
```suggestion
Tesseract supports **various output formats**: plain-text, ALTO (XML), hOCR (HTML), PDF, invisible-text-only PDF, TSV.
```

**Comment by stweil** [comment]
@jakesebright, thank you for more work on ALTO support.

I still hesitate to announce full ALTO support, because the code currently does not output `<SP>` tags.

**Comment by amitdo** [comment]
The `master` branch is the development branch, and by definition contains experimental code.

You can say: 

>Tesseract supports **various output formats**: plain-text, hOCR (HTML), PDF, invisible-text-only PDF, TSV. The `master` branch also has **experimental** support for ALTO (XML) output.

**Comment by amitdo** [comment]
See also:
https://github.com/tesseract-ocr/tesseract/wiki/ReleaseNotes#changes-made-since-last-release

File: README.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ and GitHub's log of [contributors](https://github.com/tesseract-ocr/tesseract/gr
 
 Tesseract has **unicode (UTF-8) support**, and can **recognize more than 100 languages** "out of the box".
 
-Tesseract supports **various output formats**: plain-text, hocr(html), pdf, tsv, invisible-text-only pdf.
+Tesseract supports **various output formats**: hOCR (HTML), invisible-text-only PDF, PDF, TSV. The master branch also has experimental support for ALTO (XML) output.
```
Comment (stweil) [inline README.md]:
```suggestion
Tesseract supports **various output formats**: plain text, hOCR (HTML), PDF, invisible-text-only PDF, TSV. The master branch also has experimental support for ALTO (XML) output.
```

File: doc/tesseract.1.asc [hunk 3]
```diff
@@ -90,15 +90,16 @@ OPTIONS
 	contains a list of variables and their values, one per line, with a
 	space separating variable from value.  Interesting config files
 	include: +
-  * `hocr` - Output in hOCR format (file extension `.hocr`).
-  * `pdf` - Output PDF (file extension `.pdf`).
-  * `tsv` - Output TSV (file extension `.tsv`).
-  * `txt` - Output plain text (file extension `.txt`).
+  * `alto` - Output in ALTO format (file extension `.xml`).	
   * `get.images` - Write images.
+  * `hocr` - Output in hOCR format (file extension `.hocr`).
```
Comment (stweil) [inline doc/tesseract.1.asc]: I'd keep the "Output" config files together, not mix them with config files which are rarely used.

**Comment by stweil** [comment]
@jakesebright, `<SP>` support was now added to Git master. There are still two open suggestions for the documentation, and there is a newly detected [bug in the ALTO code](https://github.com/tesseract-ocr/tesseract/blob/master/src/api/altorenderer.cpp#L152).

**Pull request merged**